docs/formats/cityscapes_user_manual.md

```diff
@@ -0,0 +1,176 @@
+# Cityscapes user manual
+
+## Contents
+
+- [Format specification](#format-specification)
+- [Load Cityscapes dataset](#load-Cityscapes-dataset)
+- [Export to other formats](#export-to-other-formats)
+- [Export to Cityscapes](#export-to-Cityscapes)
+- [Particular use cases](#particular-use-cases)
+
+## Format specification
+
+Cityscapes format overview available [here](https://www.cityscapes-dataset.com/dataset-overview/).
+Cityscapes format specification available [here](https://github.com/mcordts/cityscapesScripts#the-cityscapes-dataset).
+
+Cityscapes dataset format supports `Masks` (segmentations tasks) annotations.
+
+##  Load Cityscapes dataset
+
+The Cityscapes dataset is available for free [download](https://www.cityscapes-dataset.com/downloads/).
+
+There are two ways to create Datumaro project and add Cityscapes dataset to it:
+
+``` bash
+datum import --format cityscapes --input-path <path/to/dataset>
+# or
+datum create
+datum add path -f cityscapes <path/to/dataset>
+```
+
+It is possible to specify project name and project directory run
+`datum create --help` for more information.
+
+Cityscapes dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+└─ Dataset/
+    ├── imgsFine/
+    │   ├── leftImg8bit
+    │   │   ├── <split: train,val, ...>
+    │   │   |   ├── {city1}
+    │   │   │   |   ├── {city1}_{seq:0>6}_{frame:0>6}_leftImg8bit.png
```

Comment:
The syntax `0>6` is confusing, could you try to make it more conventional? Something like `[0...6]` etc.

Reply:
It was taken from official docs. Updated